BOJ/4153/dkdlelk99.py

```diff
@@ -0,0 +1,13 @@
+# 직각 삼각형 확인
+condition = True
+while condition:
+    a,b,c = map(int, input().split())
+    if a == 0 and b == 0 and c == 0:
+        break
+    a,b,c = a**2, b**2, c**2
```

Comment:
리뷰 예시))


변수 이름이 알기 어렵네요. 이런 식으로 바꿔보는 것은 어떤가요?
```suggestion
    x,y,z = map(int, input().split())
    if x == 0 and y == 0 and z == 0:
        break
    x,y,z = x**2, y**2, z**2
```